docs/Consensus-Protocols/QuorumIBFT.md

```diff
@@ -0,0 +1,10 @@
+description: Pantheon consensus protocols
+<!--- END of page meta data -->
+
+# IBFT 1.0 
+
+Pantheon nodes can participate (submit transactions and receive blocks) in a Quorum [IBFT 1.0](https://github.com/ethereum/EIPs/issues/650) 
+network but cannot be a validator. 
+
+To connect to a Quorum IBFT 1.0 network, use the [`--genesis-file`](../Reference/Pantheon-CLI-Syntax.md#genesis-file) 
```

Comment:
its worth noting that Quorum uses "Istanbul", whereas we use "ibft" - thus there needs to be a manual modification of Quorum's genesis file in order for us to connect (this is kinda dumb, and maybe we should spend 2 Story Points just to make this the same).
Also - I am not sure if our config params line up with Quorums, so that may require further manual mods.

Reply:
Is that a way to say that it's not going to work @rain-on ? Because if it's broken, we can't publish it as is.

Reply:
@NicolasMassart - have tested this and the only modification needed is the 'istanbul' -> 'ibft' one. Included in docs.